cmd/notation/sign.go

```diff
@@ -35,6 +35,28 @@ func signCommand(opts *signOpts) *cobra.Command {
 	command := &cobra.Command{
 		Use:   "sign [reference]",
 		Short: "Signs artifacts",
+		Long: `
+** This command is in preview and under development. **
+Prerequiste: a signing key needs to be configured using the command "notation key".
+
+Example - Sign a container image using the default signing key, with the default JWS envelope:
+  notation sign <registry>/<repository>:<tag>
+
+Example - Sign a container image using the default signing key, with the COSE envelope:
+  notation sign <registry>/<repository>:<tag> --envelope-type cose
+
+Example - Sign a container image using the specified key name
+  notation sign --key <key_name> <registry>/<repository>:<tag>
+
+Example - Sign a container image using a local key and certificate file directly
+  notation sign --key-file <key_path> --cert-file <cert_path> <registry>/<repository>:<tag>
```

Comment:
We dont support singing with local keys. Support for signing with local keys was only for test purpose.

Reply:
Yes, this command line is used for test purposes and it is available in Alpha.4. So I update it to `Sign a container image using a local testing key and certificate file directly`.